core/adminserver/admin_server.go

```diff
@@ -75,21 +75,21 @@ var (
 	}
 )
 
-// QueueAdmin controls the lifecycle and scaling of mutation queues.
-type QueueAdmin interface {
-	// AddShards creates and adds new shards for queue writing to a domain.
-	AddShards(ctx context.Context, domainID string, shardIDs ...int64) error
+// LogsAdmin controls the lifecycle and scaling of mutation logs.
+type LogsAdmin interface {
+	// AddLogs creates and adds new logs for writing to a domain.
+	AddLogs(ctx context.Context, domainID string, logIDs ...int64) error
```

Comment:
Good question. 

1. This is a bit of a convenience method. ie. whatever the caller would do, it would be nice for it to do it internally. 
2. Pavel pointed this out as well - and we concluded that retrying this method should be idempotent.